src/main/java/com/example/LoyaltyCard.java

```diff
@@ -0,0 +1,29 @@
+package com.example;
+
+import static com.google.common.base.Preconditions;
+
+
+/**
+ * Class implements Value Object.
+ */
+public class LoyaltyCard {
+    /**
+     * Customer's id.
+     */
+    private final String id;
+
+    /**
+     * Constructor for checking id with method checkNotNull.
+     *
+     * @param id is unique number
```

Comment:
This is the place to document non-nullity. Something like:

```java
  @param number a card number unique for a shop, must be a sequence of ?? number characters
```

Replace `??` with the length of the sequence. BTW, the type passed to the constructor could be `CharSequence` instead of `String`.